streams/iterate_reader.ts

```diff
@@ -35,24 +38,15 @@ export type { Reader, ReaderSync };
  * }
  * ```
  *
- * @deprecated (will be removed after 1.0.0) Use {@linkcode ReadableStreamDefaultReader} instead.
+ * @deprecated (will be removed in 1.0.0) Use {@linkcode iterateReader} instead.
```

Comment:
Maybe we should mention about the new path to import?

Reply:
Done